Mobile/starter_project_dartsmiths/lib/onboarding/presentation/screen/onboarding.dart

```diff
@@ -0,0 +1,117 @@
+import 'package:flutter/material.dart';
+
+import '../../../core/utils/colors.dart';
+import '../../../core/utils/images.dart';
+import '../widgets/onboarding_box.dart';
+import '../widgets/onboarding_button.dart';
+import '../widgets/slide_dots.dart';
+
+class Onboarding extends StatefulWidget {
```

Comment:
Rename the file and the class name to onboarding_page and OnboardingPage respectively